extensions/amp-animation/0.1/web-animations.js

```diff
@@ -306,15 +335,15 @@ export class MeasureScanner extends Scanner {
 
     // Validate.
     if (this.validate_) {
-      user().assert(!isNaN(duration),
+      user().assert(duration >= 0,
```

Comment:
you may want to call `isFiniteNumber()` (from `types.js`) for all of these numbers first ( since for instance `'' >= 0 is true`